src/main/webapp/index.html

```diff
@@ -42,14 +42,30 @@ <h3>Want to view a recommendation?</h3>
         </div>
       </div>
       <div class="map-category-section">
-        <button class="map-category-button">Restaurants</button>
-        <button class="map-category-button">Places to Visit</button>
-        <button class="map-category-button">Bars and Clubs</button>
-        <button class="map-category-button">Study Places</button>
+        <button class="map-category-button" id="restaurants-button">Restaurants</button>
+        <button class="map-category-button" id="places-button">Places to Visit</button>
+        <button class="map-category-button" id="bars-and-clubs-button">Bars and Clubs</button>
+        <button class="map-category-button" id="study-button">Study Places</button>
       </div>
     </section>
-    <!-- The Recommendation Map -->
-    <div id="map"></div>
+    <div class="wrapper">
+      <!-- The Recommendation Map -->
+      <div id="map"></div>  
+      <!-- View Recommendation Area -->
+      <div id="rec-container">
+        <p id="category-header">- Places to Visit -</p>
+        <h3>Sky Garden</h3>
+        <h4>Location / Address</h4>
+        <p id="rec-address">51.510881° N, -0.083751° W</p>
+        <h4>Description</h4>
+        <p>Sky Garden is a great place to visit, with really good
+          views of London. You can enjoy the 360 degree view
+          of the city - for free! They are public gardens and 
+          definitely worth a visit when you have the time.
+          Remember to keep socially distanced!
+        </p>
```

Comment:
I would suggest to try  using script to insert this section (e.g. inserting user card in userdir) so that the next implementation will only involve the server side, but you are free to this later :D

Reply:
Yes, as we discussed, this is the next step but at this stage, the importance was on just getting a sample content hard-coded in.